app/src/main/java/mozilla/lockbox/LockboxApplication.kt

```diff
@@ -130,4 +149,24 @@ open class LockboxApplication : Application() {
         }
         return device == "robolectric" && product == "robolectric"
     }
+
+    private class AdjustLifecycleCallbacks : ActivityLifecycleCallbacks {
+        override fun onActivityResumed(activity: Activity) {
+            Adjust.onResume()
+        }
+
+        override fun onActivityPaused(activity: Activity) {
+            Adjust.onPause()
```

Comment:
Let's check with @sandysage about what's useful for marketing (do we need `onPause`?). If we're only tracking for the install / app open, we can just use `onActivityCreated`. 

Also, I'd like to see the `AdjustLifecycleCallbacks` class in its own file; the `LockboxApplication` can get bloated quickly!

Reply:
`onResume()` and `onPause()` were part of the initial instructions to integrate this. I'm not sure if we _need_ to implement these or if they can be skipped.

Reply:
Yeah, I checked out those instructions.... I don't think we need to cargo-cult those pieces from the Adjust folks. @sandysage do you have an idea about what metrics we're trying to gather from the integration? Just installs? That will help guide which of these methods is going to be useful.